docs/reference-implementation/Brownfield-Considerations.md

```diff
@@ -0,0 +1,43 @@
+
+## Existing networking infrastructure
+
+The Azure Mission-Critical online reference implementation maintains its own virtual networks and subnets. It is supposed to be deployed fully independent of any existing infrastructure. For environments with existing networking infrastructure or requirements for connectivity to existing services either on-premises or in another virtual network, it is recommended to use the [Azure Mission-Critical - Connected](https://github.com/Azure/Mission-Critical-Connected/) reference implementation.
+
+Considerations when using existing networking infrastructure:
+
+- Prefer using individual virtual networks and subnets for each solution rather than existing, shared ones. If interconnection is required evaluate the use of Virtual Network Peerings or Private Endpoints. This is recommended to avoid potential issues due to changes made to the shared infrastructure, noisy neighbor scenarios etc.
+- For shared networks do upfront planning to avoid overlapping IP address ranges or exhaustion.
+
+## Existing container registry
+
+All Azure Mission-Critical reference implementations contain dedicated Azure Container Registries per environment. They are deployed as part of the global services and replicated to each of the regional deployment stamp locations, to host container images built and pushed as part of the overall deployment pipelines. This is done to keep the external dependencies of a deployment as minimal as possible and it also allows us to deploy the whole solution end-to-end as part of the E2E deployment pipeline.
+
+In brownfield environments it is often times considered to use an already existing central container registry for some or all container images. When a central container registry is used following considerations should be taken into account:
+
+- The container registry should be configured to replicate the images to the regional deployment stamp locations. The availability and performance of the registry can limit the overall deployments availability.
+- The compute platform used to host containers (AKS, AppServices, ..) needs to be configured to use the container registry and have permissions for pulling images. Otherwise `ImagePullSecrets` need to be specified.
+- The build/push process for container images is currently embedded into the deployment pipelines. This can be changed and separated into individual pipelines. The deployment pipeline then needs to refer to the container registry hosting the images and pointing to the right image version (using `latest` is not recommended).
+- Helm charts are currently applied from the git repository. These charts can also be pushed to a container registry and pulled from there.
+
+Reasons for a dedicated container registries per environment are:
```

Comment:
Should this turned into a recommendation? (move the **Important** remark up and only focus on it)

Reply:
agree, I think we can shorten this section and just make it a clear recommendation:
"your company might use a common/shared container registry into which application teams push their images. Thats all good and fine. Keep doing that.
But: For your MC workload, still deploy your own ACR for reasons a,b,c and import the images from that shared CR during the deployment"